### PR TITLE
Info system

### LINKED ISSUE #2: Set up workflows for testing.
We need to set up the github action test to ensure the demo is up to date with our main repo. 

We dispatch based on the Forte repo, using tools like: https://github.com/marketplace/actions/dispatch-action

### PR DESCRIPTION
**Pull request by jzpang**
Add pylint and test workflow
Fix style and pylint problem

### CODE REVIEW

File: composable_source/readers/med_mentions_reader.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def _parse_pack(self, collection: str) -> Iterator[DataPack]:
         logging.info("Processing %s.", collection)
         doc = open(collection, "r", encoding="utf8")
         text, pack_name = "", ""
+        pack: DataPack = DataPack()
```
Comment (hunterhector) [inline composable_source/readers/med_mentions_reader.py:68]: It seems the pack is created twice. Do we have to do this?

**Review by hunterhector — COMMENTED** [review]
I have one comment.

File: composable_source/readers/med_mentions_reader.py (hunk 1 above)
Comment (jzpang) [reply composable_source/readers/med_mentions_reader.py:68]: I checked the sample test file, this line is needed. In this function it is trying to create multiple datapacks, the logic is in https://github.com/petuum/composing_information_system/blob/main/composable_source/readers/med_mentions_reader.py#L69-L77

**Review by hunterhector — APPROVED** [review]
lgtm

**Pull request merged**